Gomfile

```diff
@@ -1,4 +1,5 @@
 gom 'github.com/codegangsta/negroni', :commit => 'c64702ca03d0f858ab7866638b69e85f0d4c6ee7'
+gom 'github.com/gorilla/mux', :commit => 'e444e69cbd2e2e3e0749a2f3c717cec491552bbf'
```

Comment:
`gorilla/mux` also depends on `gorilla/context`, so we should pin that here as well.

This sort of thing wouldn't be an issue if we were using godep instead of gom, but we need to figure out how to handle the `GOPATH` properly before we can switch.